q2_cutadapt/tests/test_trim.py

```diff
@@ -174,7 +173,11 @@ def test_build_trim_command_typical(self):
                                       match_read_wildcards=True,
                                       match_adapter_wildcards=False,
                                       minimum_length=2,
-                                      discard_untrimmed=True)
+                                      discard_untrimmed=True,
+                                      max_expected_errors=1,
+                                      quality_cutoff_3end=20,
+                                      quality_cutoff_5end=20,
+                                      quality_base=33)
```

Comment:
**Test:**
I would actually leave these off and then assert the defaults are in the command as expected.

(As an aside, using the same values for 3end and 5end makes the test not super helpful, since we can't determine the ordering.)